bin/ci-test.sh

```diff
@@ -2,6 +2,8 @@
 
 # test TiddlyWiki5 for tiddlywiki.com
 
+npm install
```

Comment:
In line 15 there is `npm install playwright ...` which is also added to package.json, which is installed here. -- I think line 15 is redundant